src/alto-ui/Datagrid/components/DatagridCell/DatagridCell.js

```diff
@@ -93,7 +93,7 @@ class DatagridCell extends React.Component {
     const value = this.getValue();
     const type = getType(value, column);
     const format = getFormatter(context, type);
-    return format(value, column, row);
+    return type !== 'hiddenResults' ? format(value, column, row) : '';
```

Comment:
I think that this condition shouldn't be checked here. This is too specific for alto